glusterapi/peer.py

```diff
@@ -8,10 +8,10 @@
 class PeerApis(BaseAPI):
     def peer_add(self, host, metadata=None, zone=""):
         """
-        Gluster Peer Add
+        Gluster Peer Add.
```

Comment:
sorry missed checking this one.
you need to add function document for metadata and zone also